docker-compose.yml

```diff
@@ -197,5 +198,6 @@ services:
       target: build-cron
     environment:
       NGINX_HOST: ${NGINX_SERVICE_PORT}
+      NGINX_SERVICE_PORT: ${NGINX_SERVICE_PORT}
```

Comment:
Do we need to add `NGINX_SERVICE_PORT: "http://localhost:8080"` to `deploy/(demo|development|staging)/config.yml`

Reply:
This was to solve a local problem where the NGINX_SERVICE_PORT (that is present in K8S) is available for the CRON to work

Reply:
So basically, it is already in the environment 😄